data_diff/diff_tables.py

```diff
@@ -162,7 +161,7 @@ class TableDiffer:
     """
 
     bisection_factor: int = 32  # Into how many segments to bisect per iteration
-    bisection_threshold: int = 1024**2  # When should we stop bisecting and compare locally (in row count)
+    bisection_threshold: int = 1024 * 16  # When should we stop bisecting and compare locally (in row count)
```

Comment:
Sure, don't have a huge opinion on this until we benchmark this on some remote hosts, but I feel much better about it than 1M